pkg/prometheus/promcfg_test.go

```diff
@@ -4959,6 +4959,13 @@ func TestScrapeConfigSpecConfig(t *testing.T) {
 			scSpec: monitoringv1alpha1.ScrapeConfigSpec{},
 			golden: "ScrapeConfigSpecConfig_Empty.golden",
 		},
+		{
+			name: "explicit_job_name",
```

Comment:
we need another test with a ScrapeConfig spec that defines a non-empty `relabelings` list.

Reply:
does commit #ae5a4ad8 suit you? It's not the most elegant solution but works ...

Reply:
yes, perfect!